frontend/src/pages/application_page/applicationPage.styled.js

```diff
@@ -0,0 +1,91 @@
+import { styled } from "@mui/material/styles";
+import { Container, Box, Card, CardMedia, Typography, ToggleButton, TextField, Button } from "@mui/material";
+import { NavLink } from "react-router-dom";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+
+export const CampaignImageCard = styled(Card)({
+  width: "100%",
+  borderRadius: "12px",
+  marginTop: "50px",
+  boxShadow: 20
+});
+
+export const CampaignDescription = styled(Box)({
+  width: "100%",
+  textAlign: "center",
+  paddingTop: "50px",
+  paddingBottom: "50px"
+});
+
+export const UserInfo = styled('div')({
+  display: "flex"
+});
+
+export const UserInfoFields = styled('div')({
+  width: "15%",
+  float: "left"
+});
+
+export const UserInfoTypography = styled(Typography)({
+  fontSize: "20px",
+  padding:"10px"
+});
+
+export const Link = styled(NavLink)({
+  textDecoration:"none"
+});
+
+export const FormContainer = styled(Container)`
+  padding-left: 200px;
+  padding-right: 200px;
+`;
+
+export const Section = styled("div")(({ isHidden }) => ({
```

Comment:
`styled.div(({ is Hidden...` as well (also stick to either single quotes or double quotes, not both aha)

Reply:
Should we have an eslint rule to enforce consistent quotes? https://eslint.org/docs/rules/quotes

Although it seems from [our eslintrc](https://github.com/csesoc/chaos/blob/main/frontend/.eslintrc.json) that prettier is intended to be used, which should handle this anyway 🤔